db_client/alembic/versions/0045_copy_document_role_into_metadata.py

```diff
@@ -27,6 +29,12 @@ def upgrade():
 WHERE family_document.import_id = subquery.import_id
 AND family_document.valid_metadata IS NULL"""
     )
+    op.alter_column(
+        "family_document",
+        "valid_metadata",
+        existing_type=postgresql.JSONB(astext_type=sa.Text()),
+        nullable=False,
+    )
 
 
 def downgrade():
```

Comment:
Surely there should be a way to downgrade from this as if the column has values at all rows from nullable = false then when downgrading to nullable = true should have no invalid rows? 

Or are we just not adding any downgrade functions now?

Reply:
No downgrades until we squash everything after the corpus work has finished now I believe based on the migrations chat we had last week

Reply:
There is a way to downgrade, but I don't believe we want to!

Reply:
Okay copy